pdf4qtbin.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+#All this does is just exec whatever is the argument for the graphical program to be ran.
+exec /app/bin/$1
```

Comment:
this doesn't pass the arguments.
Also is this necessary?

Reply:
I wasn't sure how to have multiple binaries in the same flatpak, so I decided to try to make a shell wrapper that would be the entry point. Do you know a better way?

Reply:
the default executable is set in `command`. I don't understand why you need the wrapper script.

Reply:
I need a bit of help with this one. I'm not sure what to make the default executable with pdf4qt. The available binaries are: UnitTests, PdfTools, Pdf4QtViewerProfi, Pdf4QtViewerLite, Pdf4QtDocPageOrganizer, and Pdf4QtDocDiff. Which one should I make the default?

Reply:
you tell me. I don't know your app.